scripts/partner_batch_imports.py

```diff
@@ -173,6 +173,14 @@ def csv_to_ol_json_item(line):
     b = Biblio(data)
     return {'ia_id': b.source_id, 'data': b.json()}
 
+def is_low_quality_book(book_item):
+    """check if a book item is of low quality"""
+    book_title_breakup = book_item.title.split()
+    if "Notebook" in book_title_breakup and book_item.publisher == "Independently Published":
```

Comment:
You should test the substring in a case-insensitive way,as I believe solr also does
```suggestion
    if "notebook" in book_title.casefold() and "independently published" in book_item.publisher.casefold():
```

Reply:
changed the code, now the test is case-insensitive. I assumed you meant book_item.title (book_item is the argument to the function is_low_quality_book)